### PR TITLE
[FEATURE] - Add an option to raise an exception if the data refresh fails

### PR DESCRIPTION
**Pull request by burkematthew**
As requested by @felciano , we have added handling so that users can request to exit prematurely if the data sync times out.  This PR includes an implementation for the Command Line as well.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -268,6 +269,7 @@ Run it as a sub-process from your favorite language; `pip install mintapi` creat
       --limit               Number of records to include from the API.  Default is 5000.
       --show-pending        Retrieve pending transactions.
                             Used with --transactions
+      --fail-if-stale       At login, Mint attempts to refresh your data.  If you wish to exit when the fail syncs, use this option.
```
Comment (mrooney) [inline README.md:272]: Should this say "when the sync fails"? Or, am I just reading it poorly?

**Review by mrooney — CHANGES_REQUESTED** [review]
Nice approach, looks great! Just one QQ.

File: README.md (hunk 1 above)
Comment (burkematthew) [reply README.md:272]: Yup, I thought I caught all of those instances.

**Review by mrooney — APPROVED** [review]
LGTM! I'm curious if the original request was to have a specific exit code that would be unique to this error, but if so, that's very easy to fix in a follow-up :)

**Pull request merged**